src/tests/utils/setup.ts

```diff
@@ -58,12 +71,17 @@ beforeAll(async () => {
   await runTenantMigrations(migrationsPath, postgresContainer.getConnectionUri());
   await insertData(orm);
 
+  global.DBexecute = async function (request_for_execute:string, db: SequelizeOrm =orm): Promise<void> {
```

Comment:
Let's declare it as 
```
global.db = {
  execute: (sqlString) => {
    // ...
  }
}
```

so in tests code we will be able to call it as
```
global.db.execute(sql)
```

Reply:
what's wrong with my realization, please explain me ?